content/en/about/integration.md

```diff
@@ -0,0 +1,57 @@
+---
+type: docs
+category: About Sigstore
+description: Integration
+home: true
+title: Integration
+weight: 3
+---
+
+![Sigstore](sigstore-logo_horizontal-color.svg)
+
+## Integration
+Many of the recent high-profile software attacks that have alarmed open-source users globally were consequences of supply chain integrity vulnerabilities: attackers gained control of a build server to use malicious source files, inject malicious artifacts into a compromised build platform, and bypass trusted builders to upload malicious artifacts. Sigstore provides a way to sign, verify, and log artifacts to ensure that they are from who they say they are and to shore up supply chain vulnerabilities.  
+
+Integrating Sigstore with your own applications provides an effective way to enhance security:
+
+- The ability to control how Sigstre is used.
```

Comment:
Let's list a few more reasons, motivating using Sigstore too. Package integrity, not needed key management, transparency, etc

Reply:
added a couple more reasons as requested